tasty-query/shared/src/main/scala/tastyquery/Definitions.scala

```diff
@@ -45,13 +45,13 @@ final class Definitions private[tastyquery] (ctx: Context, rootPackage: PackageS
   val ObjectType: TypeRef = TypeRef(javaLangPackage.packageRef, typeName("Object"))
 
   val ArrayTypeUnapplied: TypeRef = TypeRef(scalaPackage.packageRef, typeName("Array"))
-  def ArrayTypeOf(tpe: Type): AppliedType = AppliedType(ArrayTypeUnapplied, List(tpe))
+  def ArrayTypeOf(tpe: TypeOrWildcard): AppliedType = AppliedType(ArrayTypeUnapplied, List(tpe))
 
   val SeqTypeUnapplied: TypeRef = TypeRef(scalaCollectionImmutablePackage.packageRef, typeName("Seq"))
-  def SeqTypeOf(tpe: Type): AppliedType = AppliedType(SeqTypeUnapplied, List(tpe))
+  def SeqTypeOf(tpe: TypeOrWildcard): AppliedType = AppliedType(SeqTypeUnapplied, List(tpe))
 
   val RepeatedTypeUnapplied: TypeRef = TypeRef(scalaPackage.packageRef, tpnme.RepeatedParamClassMagic)
-  def RepeatedTypeOf(tpe: Type): AppliedType = AppliedType(RepeatedTypeUnapplied, List(tpe))
+  def RepeatedTypeOf(tpe: TypeOrWildcard): AppliedType = AppliedType(RepeatedTypeUnapplied, List(tpe))
```

Comment:
does this really have a wildcard sometimes?

Reply:
It's not supposed to. Unfortunately, the two call sites of this method translate from `AppliedType`s of `Seq`s or `Array`s, so statically we don't know that it's not a wildcard. Since the rest of the system would not break with a wildcard here, I consider it safer to allow it.